tests/functional/restart/42-auto-restart-ping-pong.t

```diff
@@ -38,7 +38,6 @@ BASE_GLOBAL_CONFIG='
 TEST_DIR="$HOME/cylc-run/" init_suite "${TEST_NAME_BASE}" <<< '
 [cylc]
     [[events]]
-        abort if any task fails = True
```

Comment:
I can't see a matching `--abort-if-any-task-fails` flag in this test.

Reply:
Quite right.